### PR TITLE
New landing page for /web/

### PR DESCRIPTION
**Pull request by petele**
What's changed, or what was fixed?
- Adds new landing page for /web/
- Updates architecture to support new /web/ landing page
- Improved consistency for DevSite pages

New page is staged at https://pr-5631-dot-web-central.appspot.com/web/

**Target Live Date:** 2018-01-12

- [ ] This has been reviewed and approved by (NAME)
- [X] I have run `gulp test` locally and all tests pass.
- [X] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/_index.yaml [hunk 1]
```diff
@@ -5,219 +5,230 @@ landing_page:
...
       - name: data-label
-        value: hero-updates
-
-
+        value: fundamentals-https
+      - name: data-action
+        value: web-home
+      buttons:
+      - label: Get HTTPS
+        path: /web/fundamentals/instant-and-offline/
```
Comment (ebidel) [inline src/content/en/_index.yaml]: /web/fundamentals/security/

File: src/content/en/_index.yaml [hunk 2]
```diff
@@ -5,219 +5,230 @@ landing_page:
...
+        value: webFu
       - name: data-label
-        value: solution-pwa
-    - heading: Accelerated Mobile Pages
+        value: pwa-landing
+      - name: data-action
+        value: web-home
+      buttons:
+      - label: Get Started
+        path: /web/fundamentals/instant-and-offline/
```
Comment (ebidel) [inline src/content/en/_index.yaml]: /web/progressive-web-apps/

**Comment by ebidel** [comment]
https://pr-5631-dot-web-central.appspot.com/web/progressive-web-apps/ also looks messed up.

**Comment by petele** [comment]
@ebidel - re pwa page, yah, our templating system is broken for that page. i'm working on a fix that will significantly improve our rendering.

**Pull request merged**